test/cypress/integration/card/frontend-state-cookie-removed.spec.js

```diff
@@ -0,0 +1,67 @@
+const cardPaymentStubs = require('../../utils/card-payment-stubs')
+
+describe('Frontend state cookie removed', () => {
```

Comment:
happy for this test to be removed if deemed unnecessary

Reply:
➕ I believe this is already covered in `test/controllers/secure_controller_test.js`